src/pages/TeamSettings/Service-Accounts.vue

```diff
@@ -0,0 +1,198 @@
+<script>
+import { mapGetters } from 'vuex'
+
+import Alert from '@/components/Alert'
+import ConfirmDialog from '@/components/ConfirmDialog'
+import ManagementLayout from '@/layouts/ManagementLayout'
+import ServiceAccountsTable from '@/pages/TeamSettings/Service-Accounts-Table'
+
+export default {
+  components: {
+    Alert,
+    ConfirmDialog,
+    ManagementLayout,
+    ServiceAccountsTable
+  },
+  data() {
+    return {
+      // Alert data
+      alertShow: false,
+      alertMessage: '',
+      alertType: null,
+
+      // Dialogs
+      dialogAddServiceAccount: false,
+
+      // Inputs
+      searchInput: '',
+      serviceAccountNameInput: null,
+
+      // Forms
+      serviceAccountFormValid: true,
+
+      accountCreationError: null,
+
+      // Input rules
+      rules: {
+        required: value => !!value || 'This field is is required.'
+      },
+
+      // Loading states
+      isCreatingServiceUser: false,
+
+      // Signal passed as prop to ServiceAccountsTable
+      // MembersTable watches this data attribute & refetches members every time this value changes.
+      serviceAccountsSignal: 0
+    }
+  },
+  computed: {
+    ...mapGetters('tenant', ['tenant']),
+    ...mapGetters('user', ['user']),
+    ...mapGetters('license', ['license']),
+    isTenantAdmin() {
+      return this.tenant.role === 'TENANT_ADMIN'
+    }
+  },
+  watch: {
+    tenant() {
+      this.serviceAccountsSignal++
+    }
+  },
+  methods: {
+    handleAlert(type, message) {
+      this.alertType = type
+      this.alertMessage = message
+      this.alertShow = true
+    },
+    async addServiceAccount() {
+      this.isCreatingServiceUser = true
+      this.accountCreationError = null
+
+      const res = await this.$apollo
+        .mutate({
+          mutation: require('@/graphql/User/create-service-account.gql'),
+          variables: {
+            tenant_id: this.tenant.id,
+            name: this.serviceAccountNameInput
+          }
+        })
+        .catch(({ graphQLErrors }) => {
+          this.accountCreationError =
+            'There was an error in creating your service account. Try again?'
```

Comment:
It would be more useful to show the error message as part of the error message. (Both for the user to resolve the issue and if they need to report it to us.)

Reply:
I've added it, but feel like there's probably a better way to show the error. How do I see what errors are possible, to be able to make them look nice when they occur?